oak-segment-azure/src/test/java/oak/apache/jackrabbit/oak/segment/azure/tool/SegmentCopyTestBase.java

```diff
@@ -69,7 +82,7 @@
 
     protected abstract SegmentNodeStorePersistence getDestPersistence() throws Exception;
 
-    protected abstract String getSrcPathOrUri();
+    protected abstract String getSrcPathOrUri() throws URISyntaxException, InvalidKeyException, StorageException, Exception;
```

Comment:
I suggest to remove all these exceptions here and and modify `getAzurePersistencePathOrUriSas()`to catch all exceptions and rethrow a RuntimeException.